app/Alexa/Models/Team.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Alexa\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Team extends Model
+{
```

Comment:
Need to add `SoftDeletes` trait.

Take a look at https://laravel.com/docs/5.5/eloquent#soft-deleting